boards/beagle/beagleplay_cc1352/beagleplay_cc1352.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: beagleplay_cc1352
+name: BeaglePlay CC1352
+type: mcu
+arch: arm
+ram: 144
+flash: 704
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - gpio
```

Comment:
please complete this list to increase test coverage, ex. `rtc`, `netif:openthread`, `hwinfo`, ...

Reply:
I have added hwinfo since it is present for beagleconnect_freedom as well. Not sure what else should be added. Are all the possible values documented somewhere? I have not tested openthread so that probably should not be added.